src/assets/constants/colors.ts

```diff
@@ -5,32 +5,52 @@ export const reviewColors = {
 };
 
 export const coursesColorSet = [
```

Comment:
Are both `coursesColorSet` and `SUBJECT_COLORS` necessary? Can we make them 1 thing?